### ISSUE TITLE
Export compact Review decision calibration samples

### ISSUE THREAD

**Issue by pawelogrodnik** [pawelogrodnik/socca-vision-analysis#85]
## Context

PR #84 introduced coverage-driven Team-U Review and the first durable Review decision audit / benchmark pipeline.

The historical backfill for the three existing Corgi vs Verisk analyses is intentionally conservative:

- `9c7485e4`: 644 persisted operator decisions
- `6d8fc20c`: 236 persisted operator decisions
- `5e62625e`: 90 persisted operator decisions
- exact source linkage: 0 for all three
- recoverable team evidence: 0 for all three
- dominant-signal eligible cases: 0 for all three

This is correct: legacy stores preserve final canonical decisions, but not enough exact pre-click source/evidence state to reconstruct reliable calibration samples.

The next step is to export a compact, durable calibration dataset from **new live Review audit events**, without committing huge raw backfill artifacts.

## Goal

Create a compact per-decision calibration dataset that can later answer questions such as:

- For short sources with dominant Team B upstream evidence, how often did the operator also establish Team B?
- Is `SHORT_TRACK_MIN_DOMINANT_RATIO = 0.85` safe, or should it be 0.90 / 0.95?
- How does agreement change by source length, switch count, minority-run length, or sample size?
- Which Review decisions are potentially safe to automate, and which must remain operator-reviewed?

This should be treated as a **calibration dataset**, not yet as an ML training dataset.

## Required artifact

For each match, export a compact file such as:

`backend/decision-benchmarks/<match-id>/review_decision_calibration_samples.jsonl`

One JSON line = one operator decision / calibration sample.

Do not use frame-level rows as independent samples.

## Suggested sample contract

Each sample should preserve only durable fields needed for later calibration, for example:

```json
{
  "schema_version": "1.0.0",
  "match_id": "abc123",
  "event_id": "event-id",
  "provenance": "EXACT_PERSISTED",

  "source": {
    "scope_kind": "whole_subject",
    "source_frame_count": 84,
    "observation_count": 63,
    "duration_sec": 2.8
  },

  "team_evidence_before": {
    "A_observations": 4,
    "B_observations": 55,
    "U_observations": 4,
    "known_team_observations": 59,
    "dominant_team": "B",
    "dominant_ratio": 0.9322,
    "team_switch_count": 2,
    "longest_A_run": 2,
    "longest_B_run": 31
  },

  "review_context": {
    "required": true,
    "scope_policy": "whole_subject",
    "structural_conflict": false,
    "operator_contradiction": false
  },

  "operator_result": {
    "action": "assign_roster_player",
    "effective_team_label": "B"
  },

  "automatic_policy": {
    "policy_version": "short_track_dominant_team_v1",
    "would_auto_assign": true
  }
}
```

The exact schema may differ, but the semantics above must be preserved.

## Audit improvement required

The live operator audit must persist an authoritative `effective_team_label` / equivalent final team disposition for any operator action that establishes team truth.

Examples:

- `assign_team` -> selected A/B
- `assign_roster_player` -> team of the selected canonical roster player
- `assign_existing_slot` -> team of the selected stable slot
- `create_new_stable_player` -> selected A/B
- `team_unknown` -> U
- `referee` -> referee
- `false_detection` -> false_detection

This must be derived server-side from authoritative state. Do not trust an optional browser-supplied team label.

A player-identity action may therefore remain classified as a player-identity UI/action event while also contributing to team-attribution calibration if the pre-decision source actually carried a team-attribution question.

Do not double-count total operator decisions.

## Eligibility rules

Calibration samples should be emitted only when the event contains genuine persisted pre-decision evidence.

At minimum:

- provenance must be `EXACT_PERSISTED`
- event must represent a real human decision, never an automatic short-track assignment
- exact pre-decision team evidence must be available
- source identity/digest must still correspond to the event snapshot
- no historical reconstruction may be presented as exact persisted truth

If evidence is unavailable, omit the calibration sample rather than fabricating one.

Historical records may remain useful for aggregate workload statistics, but should not be promoted into dominant-signal calibration samples unless exact provenance can be proven.

## Calibration fields to preserve

The compact sample should be sufficient to evaluate at least:

- dominant team
- dominant ratio
- A/B/U observation counts
- known-team observation count
- source frame count / duration
- team switch count
- longest minority run / A+B run statistics
- Required vs Optional when known
- structural-conflict / operator-contradiction safety state
- operator final team disposition
- operator action
- current short-track policy version
- whether the current policy would have auto-assigned this sample

Keep A/B match-relative. Do not learn jersey colors from this dataset.

## Benchmark outputs

Continue to keep the compact aggregate:

`review_decision_benchmark.json`

It should be possible to derive or cross-check summary statistics from calibration samples, including:

- eligible dominant-signal sample count
- operator agreement / override count
- agreement by dominant-ratio bucket
- agreement by source-length bucket
- agreement by switch-count bucket
- agreement by minority-run bucket
- short/high-dominance summary

Future threshold changes must be based on held-out / cross-match evidence, not on one match or frame pseudo-samples.

## Git / artifact hygiene

Do not commit huge raw backfill reports.

Recommended committed structure:

```
backend/decision-benchmarks/<match-id>/
  match.json
  review_decision_benchmark.json
  review_decision_calibration_samples.jsonl
```

Raw artifacts such as:

`review_operator_decision_backfill_report.json`

should remain ignored / local unless explicitly needed for debugging.

Suggested `.gitignore` policy:

```gitignore
backend/decision-benchmarks/**

!backend/decision-benchmarks/*/
!backend/decision-benchmarks/*/match.json
!backend/decision-benchmarks/*/review_decision_benchmark.json
!backend/decision-benchmarks/*/review_decision_calibration_samples.jsonl
```

## Non-goals

- Do not lower the current Reviewed coverage thresholds.
- Do not change the accepted coverage-driven Team-U behavior from PR #84.
- Do not change YOLO/tracking.
- Do not use crop pixels as a Review-time team classifier.
- Do not auto-promote historical reconstructed data to ground truth.
- Do not train an ML model in this issue.
- Do not treat individual frames from one operator decision as independent calibration samples.

## Acceptance criteria

- New live Review decisions can produce compact calibration samples.
- `assign_roster_player` / slot / new-player actions expose authoritative final team disposition when relevant.
- Automatic short-track decisions never appear as human ground truth.
- A single operator decision produces one calibration sample.
- Missing exact pre-decision evidence results in no sample rather than guessed data.
- Compact per-match JSONL export exists.
- Aggregate benchmark remains available.
- Large raw backfill artifacts are ignored by Git.
- Tests cover A/B agreement, override, U/referee/false detection, and non-team player-identity cases that must not become calibration samples.
- Existing PR #84 Team-U / Review convergence behavior remains unchanged.
